common/src/main/scala/org/apache/comet/objectstore/NativeConfig.scala

```diff
@@ -40,6 +41,8 @@ object NativeConfig {
     // Azure Data Lake Storage Gen2 secure configurations (can use both prefixes)
     "abfss" -> Seq("fs.abfss.", "fs.abfs."))
 
+  val COMET_LIBHDFS_SCHEMES_KEY = "fs.comet.libhdfs.schemes"
```

Comment:
Should we make this a Comet conf (i.e add it in `CometConf` so it is automatically documented)?

Reply:
Thanks, moved to CometConf.